src/vm/exceptionhandling.cpp

```diff
@@ -5047,7 +5047,7 @@ BOOL IsSafeToHandleHardwareException(PCONTEXT contextRecord, PEXCEPTION_RECORD e
         exceptionRecord->ExceptionCode == STATUS_BREAKPOINT || 
         exceptionRecord->ExceptionCode == STATUS_SINGLE_STEP ||
         (IsSafeToCallExecutionManager() && ExecutionManager::IsManagedCode(controlPc)) ||
-#ifdef _TARGET_ARM_
+#if defined(_TARGET_ARM_) || (defined(_TARGET_X86_) && defined(FEATURE_PAL))
         IsIPinVirtualStub(controlPc) ||  // access violation comes from DispatchStub of Interface call
```

Comment:
You may remove FEATURE_PAL as WIN64EXCEPTIONS implies FEATURE_PAL anyway, but it seems optional.